### PR TITLE
feat(UI): App Details refactor for consistency

### PR DESCRIPTION
**Pull request by damienomurchu**
## Motivation
Follow on refactoring changes from https://github.com/aerogear/mobile-security-service/pull/91 for consistency

## What
update variable & file names 

## Why
code consistency

## How
Update variable & file names in code

## Checklist:

- [x] Code has been tested locally by PR requester
- [ ] Changes have been successfully verified by another team member 

## Progress

- [x] Finished task

### CODE REVIEW

**Comment by damienomurchu** [comment]
@laurafitzgerald almost ready, just need to correct a failing test

File: ui/src/components/App.js [hunk 1]
```diff
@@ -10,7 +10,6 @@ class App extends React.Component {
       <Router>
         <div className="App">
           <Switch>
-            import AppDetailedView from './AppDetailedView';
```
Comment (damienomurchu) [inline ui/src/components/App.js]: this was accidentally committed and merged with previous PR, and just removing here

File: ui/src/containers/AppTableContainer.js [hunk 2]
```diff
@@ -19,25 +19,25 @@ export class AppDetailsContainer extends React.Component {
...
+      tempRow[2] = this.props.app[i][2];
+      tempRow[3] = this.props.app[i][3];
       const cb = (
         <React.Fragment>
           <Checkbox
             label=""
-            isChecked={this.props.apps[i][4]}
+            isChecked={this.props.app[i][4]}
             onChange={this.handleChange}
             aria-label="controlled checkbox example"
```
Comment (laurafitzgerald) [inline ui/src/containers/AppTableContainer.js]: We probably want to change this label as well.

File: ui/src/reducers/index.js [hunk 3]
```diff
@@ -66,13 +66,13 @@ export default (state = initialState, action) => {
         }
       };
     case APP_DETAILS_SORT:
-      const newOrder = action.payload.direction;
+      const appDirection = action.payload.direction;
       const colIndex = action.payload.index;
```
Comment (laurafitzgerald) [inline ui/src/reducers/index.js]: colIndex -> appIndex

File: ui/src/containers/AppTableContainer.js (hunk 2 above)
Comment (damienomurchu) [reply ui/src/containers/AppTableContainer.js]: good spot :+1:

File: ui/src/reducers/index.js (hunk 3 above)
Comment (damienomurchu) [reply ui/src/reducers/index.js]: sure

**Comment by damienomurchu** [comment]
@laurafitzgerald done now if you can approve

**Pull request merged**